lib/chef/provider/package/rubygems.rb

```diff
@@ -401,7 +401,7 @@ def initialize(new_resource, run_context=nil)
         end
 
         def is_omnibus?
-          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef)/embedded/bin!
+          if RbConfig::CONFIG['bindir'] =~ %r!/opt/(opscode|chef|chefdk)/embedded/bin!
```

Comment:
It would also be nice to make this work on Windows machines - by default it installs to `C:\opscode\chefdk`